sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/KeyVaultCredentialPolicy.java

```diff
@@ -224,15 +322,10 @@ public String getTenantId() {
         }
     }
 
-    public static void clearCache() {
-        CHALLENGE_CACHE.clear();
-    }
-
```

Comment:
I don't think we should get rid of this. Maybe it was removed on accident? I can see it on the other 3 Key Vault policy files.

Reply:
I think I just moved it a couple of lines above here https://github.com/Azure/azure-sdk-for-java/pull/29730/files#diff-7bb81c7c946d6a6c3a465a01cc22b5b702607792a9f79d2c27d3efb3705d016aR287